### ISSUE TITLE
AdvancedBan Permissions Not Showing Up with LuckPerms

### ISSUE THREAD

**Issue by ShreyasAyyengar** [DevLeoko/AdvancedBan#356]
_What version of AdvancedBan (`/AdvancedBan`) are you using?_
 2.1.9-RELEASE.JAR
 
_What kind of server do you have (`Bungeecord/single server`)?_
 BungeeCord
 
_What server version (/version) are you using?_
This server is running CraftBukkit version git-Spigot-800b93f-8160e29 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)

_Please provide the EXACT steps required to reproduce the problem..._
1. Download LuckPerms and AdvancedBan
2. Put AdvancedBan in your BungeeCord Plugins folder (not the individual spigot servers) just the Bungeecord Plugins folder
3. Put the LuckPerms plugin in any server plugin folder you like. 
3. Lauch MC
4. Go into your BungeeCord server and do /server (Server you put luck perms in)
5. Enter the console command /lp user <your username> permission set luckperms.* true
6. Do /lp editor.
7. Open the assigned link
8. Try to do any ab. perms. (It won't autofill)
9. If it doesn't autofill, try just inputting any perm manually like ab.ban.perma
10. Click the save changes at the top of the luck perms editor, and copy and paste the command it gives you into your game (player command). It will say, 
AdvancedBan You don't have perms for that (or somthing similar) 

When I try to add a perm for AdvancedBan, I try to add a perm to luck perms web editor by just doing ab. usually the web editor will complete the perm I am trying to input, but nothing shows up for me with AdvancedBan at all. 

Even when I manually type a perm into luck perms such as ab.ban.perma, when I apply the edits, in game it still doesn't work. I can do the command from the console so I know the commands work I just don't know why perms doesn't

Advanced ban is in my bungee plugin.

While luck perms is in my spigot server. 

Any help?

Shreyas Ayyengar

**Comment by DevLeoko** [comment]
I think you also need to install LuckPerms on the BungeeCord server

**Comment by ShreyasAyyengar** [comment]
I, do, I have LuckPerms and AdvancedBan in my BungeeCord Server, with an additional luck perms plugin in my Spigot server.

Could you help me with this?

Thanks!
Shreyas Ayyengar

**Comment by DevLeoko** [comment]
Try `lp check <Player> <Permission>` in your bungeecord console. And also please send me an screenshot of the message you get when you try to do /ban

**Comment by ShreyasAyyengar** [comment]
When I do lp check shreyas007 permission on my bungee console it says command not found, however when I do it on my spigot server it says 

LP Permission Check result on user Shreyas007 for permission permission True 

When I do /ban it says AdvancedBan - You don’t have perms for that! 

![image](https://user-images.githubusercontent.com/65931801/82885174-5ab5a600-9f77-11ea-94db-2230fcfb8d90.jpg)

**Comment by DevLeoko** [comment]
This really seems like an issue with your permissions setup and not AdvancedBan. I don't know too much about LuckPerms, so I can't really help you with that.

Please use their support or join our discord server: https://discord.gg/ycDG6rS (there are a lot of people that have the same setup)

**Comment by Hopefuls** [comment]
> 
> 
> This really seems like an issue with your permissions setup and not AdvancedBan. I don't know too much about LuckPerms, so I can't really help you with that.
> 
> Please use their support or join our discord server: https://discord.gg/ycDG6rS (there are a lot of people that have the same setup)

TL;DR: User actually tried contacting us, but said to us, i quote

> if that's how ur treating the people that use ur plugins, that's fucking rude lol. get a life tho

So we were unfortunately not able to help this user after explaining what exactly the issue was 🤷

**Comment by Blocksnmore** [comment]
You need to use /lpb and not /lp unless you have them synced via mysql

**Comment by ShreyasAyyengar** [comment]
> You need to use /lpb and not /lp unless you have them synced via mysql

Ye ik that now I asked this quite some time ago xD

**Comment by Odyrac** [comment]
i have the same problem an idea ?
